cfme/tests/v2v/test_v2v_ansible.py

```diff
@@ -0,0 +1,181 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.fixtures.provider import rhel7_minimal
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+from cfme.utils.appliance.implementations.ui import navigate_to, navigator
+from cfme.utils.conf import credentials
+from cfme.utils.log import logger
+from cfme.utils.wait import wait_for
+
+
+pytestmark = [
+    test_requirements.ansible,
+    pytest.mark.meta(
+        server_roles=["+embedded_ansible"]
+    ),
+    pytest.mark.provider(
+        classes=[RHEVMProvider],
+        selector=ONE_PER_VERSION,
+        required_flags=["v2v"]
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_VERSION,
+        fixture_name="second_provider",
+        required_flags=["v2v"]
+    ),
+]
+
+REPOSITORIES = "https://github.com/v2v-test/ansible_playbooks.git"
+
+
+def get_migrated_vm_obj(src_vm_obj, target_provider):
+    """Returns migrated_vm obj from target_provider"""
+    collection = target_provider.appliance.provider_based_collection(target_provider)
+    migrated_vm = collection.instantiate(src_vm_obj.name, target_provider)
+    return migrated_vm
+
+
+@pytest.fixture(scope="module")
+def wait_for_ansible(appliance):
+    appliance.wait_for_embedded_ansible()
+
+
+@pytest.fixture(scope="module")
+def credentials_collection(appliance):
+    """Fixture to add machine credentials"""
+    return appliance.collections.ansible_credentials
+
+
+@pytest.fixture(scope="module")
+def ansible_repository(appliance):
```

Comment:
Since this depends on `wait_for_ansible`, that fixture should be a parameter here.  They're both module so it should be simple.  That way the dependency is built in, and the test won't fail if the fixture order changes in the test definition.